test/integration/logging/local_live_log_test.js

```diff
@@ -78,6 +76,23 @@ suite('live logging', () => {
       token.length,
       22, 'Token should be 22 characters long'
     );
+  });
 
+  test('live log expiration date', async () => {
+    let expiration = taskcluster.fromNowJSON('1 second');
```

Comment:
Fun fact: live logs for tests were expiring in 3.6 seconds instead of 3600 seconds=1 hour. This was a workaround that I forgot to figure out after I got the tests working.